receiver/skywalkingreceiver/README.md

```diff
@@ -40,5 +40,8 @@ service:
   pipelines:
     traces:
       receivers: [skywalking]
+    metrics:
+      receivers: [skywalking]
+      
```

Comment:
In this PR, skywalking metrics only support `jvm` metrics, right? So, we need some notes to user. It does not support skywalking meter(https://github.com/apache/skywalking-java/blob/main/docs/en/setup/service-agent/java-agent/Application-toolkit-meter.md).

Reply:
Yes, i will add some notes